src/pyosmeta/cli/update_review_contribs.py

```diff
@@ -0,0 +1,137 @@
+"""
+This script parses through our reviews and contributors and:
+
+1. Updates reviewer, editor and maintainer data in the contributor.yml file to
+ensure all packages they supported are listed there.
+1b: And that they have a listing as peer-review under contributor type
+2. Updates the packages metadata with the participants names if it's missing
+3. FUTURE: finally it looks to see if we are missing review participants from
+the review issues in the contributor file and updates that file.
+
+This script assumes that update_contributors and update_reviews has been run.
+Rather than hit any api's it just updates information from the issues.
+To run: update_reviewers
+
+# TODO - FEATURE we have some packages that were NOT approved but we had editors and reviewers.
+# We need to acknowledge these people as well. maybe tag them with waiting on maintainer response??
+# TODO: package-wide feature: create a flag for entries that we do not want to update
+# TODO: make sure we can add a 3rd or 4th reviewer - crowsetta has this as
+# will biocypher
+# TODO: make sure to add a current editor boolean to the current editors and
+# emeritus ones.
+# TODO - ?create a class for person types??
+
+"""
+
+
+import pickle
+from typing import Dict, List, Optional, Tuple, Union
+
+from pyosmeta.contributors import ProcessContributors
+from pyosmeta.file_io import clean_export_yml, get_api_token, load_pickle
+
+
+def get_clean_user(username: str):
+    return username.lower().strip()
+
+
+def main():
+    GITHUB_TOKEN = get_api_token()
```

Comment:
if you are not planning to have different kind of connections
maybe you could move this step directly to ProcessContributors.
in another words, `ProcessContributors` could get the api token directly

Reply:
that worked!